internal/sdkprovider/service/project/util.go

```diff
@@ -0,0 +1,31 @@
+package project
+
+import (
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DetermineAccountIDPointerForRequest returns the account ID for a request.
```

Comment:
it gets the account ID for the use in requests, I have updated its name to make it better, also it shouldn't be there because it only belongs to this domain